packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, useContext, useEffect, useRef, useState } from 'react';
+import * as awsui from '@awsui/design-tokens';
```

Comment:
will remove. forgot to remove it when I removed the inline styles. 
importing everything as `awsui` seems to be the pattern the awsui team recommended since this pattern is what they have in their development guideline

Reply:
Mass imports like that tend to lead to bloat & we have a lot of those in our package. It would probably benefit us to start getting more specific as to what we want. Not a major thing, but something to consider.